functions/vcs.zsh

```diff
@@ -110,7 +110,18 @@ function +vi-hg-bookmarks() {
 
 function +vi-vcs-detect-changes() {
   if [[ "${hook_com[vcs]}" == "git" ]]; then
-    vcs_visual_identifier='VCS_GIT_ICON'
+
+    local remote=`git ls-remote --get-url`
+    if [[ "$remote" =~ "github" ]] then
```

Comment:
Isn't `=~` meant for regex comparisons? For direct string-to-string comparisons, I think `==` is preferred?